.github/workflows/ci.yaml

```
@@ -203,18 +210,19 @@ jobs:
       HELM_VERSION: v3.8.1
     steps:
       - name: Checkout
-        uses: actions/checkout@v7
+        uses: actions/checkout@3d3c42e5aac5ba805825da76410c181273ba90b1 # v7
         with:
           fetch-depth: 0
           ref: ${{ needs.get_ref.outputs.ref }}
```

actionlint (1.7.12)

[error] property "get_ref" is not defined in object type {get_info: {outputs: {e2e_run: string; version: string}; result: string}; test: {outputs: {}; result: string}}

(expression)